[2018-08-09 Thu 21:27:39]fuzztask.py-929-139711149385472-34-    INFO-"Thread-5 FUZZ检测任务启动2018-08-09 21:27:39.805011"
[2018-08-09 Thu 21:27:39]fuzztask.py-929-139711148857088-34-    INFO-"Thread-6 FUZZ检测任务启动2018-08-09 21:27:39.821865"
[2018-08-09 Thu 21:27:39]fuzztask.py-929-139711148857088-49-    INFO-"FUZZ检测:   122.114.49.116:80,http"
[2018-08-09 Thu 21:27:39]fuzztask.py-929-139711149385472-49-    INFO-"FUZZ检测:   113.105.74.144:80,http"
[2018-08-09 Thu 21:28:50]fuzztask.py-929-139711140767488-34-    INFO-"Thread-13 FUZZ检测任务启动2018-08-09 21:28:50.716190"
[2018-08-09 Thu 21:28:50]fuzztask.py-929-139711140767488-49-    INFO-"FUZZ检测:   61.50.129.38:80,http"
[2018-08-09 Thu 21:28:50]fuzzdetect.py-929-139711140767488-163-   ERROR-"[InfoDisScanner._scan_worker][2][61.50.129.38:80/admin/] Exception global name 'status' is not defined"
[2018-08-10 Fri 10:56:47]fuzztask.py-13006-140682676299520-34-    INFO-"Thread-6 FUZZ检测任务启动2018-08-10 10:56:47.922272"
[2018-08-10 Fri 10:56:47]fuzztask.py-13006-140682675771136-34-    INFO-"Thread-7 FUZZ检测任务启动2018-08-10 10:56:47.944552"
[2018-08-10 Fri 10:56:48]fuzztask.py-13006-140682675771136-49-    INFO-"FUZZ检测:   61.50.129.38:80,http"
[2018-08-10 Fri 10:56:48]fuzztask.py-13006-140682676299520-49-    INFO-"FUZZ检测:   113.105.74.144:80,http"
[2018-08-10 Fri 10:56:48]fuzzdetect.py-13006-140682675771136-163-   ERROR-"[InfoDisScanner._scan_worker][2][61.50.129.38:80/admin/] Exception global name 'status' is not defined"
[2018-08-10 Fri 11:12:13]fuzztask.py-13006-140682675771136-53-    INFO-"Thread-7FUZZ检测任务结束2018-08-10 11:12:13.900269"
[2018-08-10 Fri 11:12:15]fuzztask.py-13006-140682676299520-53-    INFO-"Thread-6FUZZ检测任务结束2018-08-10 11:12:15.330265"
